test/sandbox-test.js

```diff
@@ -56,6 +56,18 @@ describe("sinonSandbox", function () {
         assert.same(sandbox.assert, sinonAssert);
     });
 
+    it("can be reset without failing when pre-configured to use a fake server", function () {
+        var sandbox = sinonSandbox.create({useFakeServer: true});
+        sandbox.reset();
```

Comment:
ah, good catch. I'll add some kind of refutation along with `throws` (don't remember the exact api).

failing test
```
 1) sinonSandbox can be reset without failing when configured to use a fake server:
     TypeError: Cannot set property 'length' of undefined
      at Object.resetBehavior (lib/sinon/util/fake_server.js:279:51)
      at Object.reset (lib/sinon/util/fake_server.js:274:14)
      at lib/sinon/collection.js:26:21
      at Array.forEach (native)
      at each (lib/sinon/collection.js:25:19)
      at Object.reset (lib/sinon/collection.js:41:9)
      at Context.<anonymous> (test/sandbox-test.js:68:17)

```